src/intel_rdt.c

```diff
@@ -325,8 +325,12 @@ static int strlisttoarray(char *str_list, char ***names, size_t *names_num) {
       continue;
 
     if ((isdupstr((const char **)*names, *names_num, token))) {
-      ERROR(RDT_PLUGIN ": Duplicated process name \'%s\' in group \'%s\'",
-            token, str_list);
+      if (str_list != NULL)
```

Comment:
I think you can simplify this by removing the `if (str_list != NULL)` statement and just leave the last line `ERROR(RDT_PLUGIN ": Duplicated process name \'%s\'", token);`, because the expression `str_list != NULL` will never be true after line 319 (`str_list = NULL;`). But your code works, so it's up to you. How do you think?

Reply:
Yes, that was what I was thinking. I'll change the PR to match our common understanding. RDT is not my code, that I why I was looking for your feedback here.